komga-webui/src/components/ThumbnailExplorerDialog.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-dialog v-model="input" scrollable>
+    <v-card :max-height="$vuetify.breakpoint.height * .9" dark>
+      <v-card-text>
+        <v-container fluid>
+          <v-row class="mb-2 d-flex flex-sm-row flex-column align-center justify-space-around">
+
+            <v-pagination
+              v-model="page"
+              :total-visible="perPage"
+              :length="Math.ceil(thumbnails.length/perPage)"
+            ></v-pagination>
+
+            <div v-for="(url, i) in visibleThumbnails()"
+                 :key="i"
+                 style="min-height: 220px; max-width: 140px"
+                 class="d-flex flex-column justify-center"
+            >
+              <v-img
+                :src="url"
+                lazy-src="../assets/cover.svg"
+                aspect-ratio="0.7071"
+                :contain="true"
+                max-height="200"
+                max-width="140"
+                class="ma-2"
+                @click="input = false; goTo(((page - 1 ) * perPage + i + 1))"
+                style="cursor: pointer"
+              />
+              <div class="white--text text-center font-weight-bold">{{ (page - 1 ) * perPage + i + 1 }}</div>
+            </div>
+
+          </v-row>
+        </v-container>
+      </v-card-text>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { bookPageThumbnailUrl } from '@/functions/urls'
+
+export default Vue.extend({
+  name: 'ThumbnailExplorerDialog',
+  props: {
+    pagesCount: {
+      type: Number
+    },
+    value: {
+      type: Boolean
+    },
+    bookId: {
+      type: Number
+    }
+  },
+  data: () => {
+    return {
+      input: '',
+      page: 1,
+      perPage: 8
+    }
+  },
+  watch: {
+    value: {
+      handler (after) {
+        this.input = after
+      },
+      immediate: true
+    }
```

Comment:
```suggestion
    value (val) {
      this.input = val
    }
```

This can be shortened